test/test_datacheck.py

```diff
@@ -7,7 +7,7 @@
 
 sys.path.append(os.path.join(os.path.split(__file__)[0], ".."))
 
-from test import BASE_DATA, BASE_WORK, LOGGER, QgisTest, iface
+from base_test import BASE_DATA, BASE_WORK, LOGGER, QgisTest, iface
```

Comment:
Big fan of moving stuff out of `__init__`, however this should read `.base_test` and be a relative import.